src/modules/setavatar.js

```diff
@@ -0,0 +1,36 @@
+const Eris = require("eris");
+const threadUtils = require("../threadUtils");
+const axios = require("axios");
+
+const validate_img = /(http(s?):)([/|.|\w|\s|-])*\.(?:jpg|gif|png)/;
+
+/**
+ * @param {Eris.CommandClient} bot
+ */
+module.exports = bot => {
+    bot.registerCommandAlias("setav");
```

Comment:
This needs to be after addInboxServerCommand

Reply:
Also, you need to specify which alias it belongs to. Refer to Eris docs for information